test/AspNetCoreModule.Test/FunctionalTestHelper.cs

```diff
@@ -35,14 +35,14 @@ public bool IsMet
         {
             get
             {
```

Comment:
These are used for an attribute to determine whether the tests is enabled or not.